pom.xml

```diff
@@ -1119,6 +1118,7 @@
 	    <exclude>**/*.html</exclude>
 	    <exclude>**/sit</exclude>
       <exclude>**/test/queries/**/*.sql</exclude>
+        <exclude>**/ql/io/parquet/timestamp/datetime/**</exclude>
```

Comment:
according to https://www.apache.org/legal/src-headers.html#3party, we should not add AL2 license header to the copied source files